components/new-contribution-flow/ContributionFlowSuccess.js

```diff
@@ -232,113 +245,161 @@ class NewContributionFlowSuccess extends React.Component {
     return <PublicMessageForm order={this.props.data.order} publicMessage={publicMessage} />;
   };
 
+  renderBankTransferInformation = () => {
+    let instructions = get(this.props.data, 'order.toAccount.host.settings.paymentMethods.manual.instructions', null);
+    const bankAccount = get(this.props.data, 'order.toAccount.host.payoutMethods[0].data', null);
+    const amount = (
+      <FormattedMoneyAmount
+        amount={
+          (get(this.props.data, 'order.amount.value') +
+            get(this.props.data, 'order.platformContributionAmount.value', 0)) *
+          100
+        }
+        currency={get(this.props.data, 'order.amount.currency')}
+      />
+    );
+    const renderInstructions = () => {
+      const formatValues = {
+        account: formatAccountDetails(bankAccount),
+        reference: get(this.props.data, 'order.id', null),
+        amount,
+        collective: get(this.props.data, 'order.toAccount.name', null),
+      };
+      return instructions.replace(/{([\s\S]+?)}/g, (match, p1) => {
+        if (p1) {
+          const key = p1.toLowerCase();
+          if (formatValues[key] !== undefined) {
+            return formatValues[key];
+          }
+        }
+        return match;
+      });
+    };
```

Comment:
**[:ice_cream: Details & personal preferences]** Could this be a class method (that you call with `this.renderInstructions(...)`) to simplify the code here? Or extract this to a new `BankTransferInstructions` (or similar) component?